github/pom.xml

```diff
@@ -45,6 +53,18 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-arc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
```

Comment:
`quarkus-oidc-client` is not needed, it is for acquiring tokens with anything but the authorization code flow grant (though it can help to implement custom code flow implementations)